plugins/cache_range_requests/cache_range_requests.cc

```diff
@@ -134,25 +133,16 @@ create_pluginconfig(int argc, char *const argv[])
   --argv;
 
   for (;;) {
-    int const opt = getopt_long(argc, argv, "i:", longopts, nullptr);
+    int const opt = getopt_long(argc, argv, "j:", longopts, nullptr);
     if (-1 == opt) {
       break;
     }
 
     switch (opt) {
-    case 'c': {
```

Comment:
the above options should be removed as well line 121 and 123

Reply:
removed